client/src/app/users/user-list.component.scss

```diff
@@ -12,7 +12,7 @@
 }
 
 .mat-radio-button {
-  margin: 0 12px;
+  margin: 0 1rem;
```

Comment:
Nice change. Do we use `px` anywhere else?

Reply:
I think we use px *everywhere* else. I tried this kind of on a whim...

Reply:
The Angular Material stuff generally uses px values for styles are least as of the last time I checked.